frontend/src/features/shared/receipt/InfoSection.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardTitle,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@freemed-kit/ui';
+import { ReceiptFormValues } from '@/features/shared/receipt/useReceiptForm';
+import { GENDER } from '@/constants/gender';
+import DatePicker from '@/components/ui/DatePicker';
+
+interface InfoSectionProps {
+  readOnly: boolean;
+}
+
+const InfoSection = ({ readOnly }: InfoSectionProps) => {
+  const form = useFormContext<ReceiptFormValues>();
+
+  return (
+    <>
+      <div>
+        <Card>
+          <CardHeader>
+            <CardTitle>기본정보</CardTitle>
+          </CardHeader>
+          <CardContent className="space-y-4">
+            <div className="grid grid-cols-4 gap-4">
+              <FormField
+                control={form.control}
+                name="name"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>이름</FormLabel>
+                    <FormControl>
+                      <Input
+                        placeholder="이름을 입력하세요"
+                        {...field}
+                        readOnly={readOnly}
+                      />
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="gender"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>성별</FormLabel>
+                    <FormControl>
+                      <Select
+                        onValueChange={field.onChange}
+                        value={field.value}
+                        disabled={readOnly}
+                      >
+                        <SelectTrigger>
+                          <SelectValue placeholder="선택" />
+                        </SelectTrigger>
+                        <SelectContent>
+                          <SelectItem value={GENDER.MALE.value}>
+                            {GENDER.MALE.label}
+                          </SelectItem>
+                          <SelectItem value={GENDER.FEMALE.value}>
+                            {GENDER.FEMALE.label}
+                          </SelectItem>
+                        </SelectContent>
+                      </Select>
+                    </FormControl>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="birth"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>날짜 선택</FormLabel>
+                    <FormControl>
+                      <DatePicker
+                        date={field.value}
+                        onChange={field.onChange}
```

Comment:
```suggestion
                        date={field.value ? new Date(field.value) : undefined}
                        onChange={(date) =>
                          date &&
                          field.onChange(
                            format(date, 'yyyy-MM-dd', { locale: ko }),
                          )
                        }
```

birth 형식에 맞춰 date, onChange는 이렇게 수정하면 되구요!